inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -311,9 +327,19 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
 
     writeBits(header, os);
 
+    for (auto &name : inputNames) {
+        const auto nameSize = strlen(name.c_str()) + 1;
```

Comment:
why don't use name.size() instead of mixing C++ and C methods?

Reply:
I've recommended to use strlen() instead of size() as c_str() is used (c_str() is more suited to C then C++).
size() may give different value in some circumstances...
https://stackoverflow.com/questions/29302073/difference-between-strlenstr-c-str-and-str-length-for-stdstring